.gitlab-ci.yml

```diff
@@ -220,6 +221,16 @@ test-rust-nightly:
     - rust
     - rust-nightly
   allow_failure: true
+json-rpc-docs:
+  stage: docs
+  only:
+    - tags
+    - master
```

Comment:
might be sufficient to have this on `tags` only.

Reply:
I agree. Having that for every commit to master would get annoying pretty quick.